### PR TITLE
[Windows] Change log output to `Testing …` when testing a SwiftPM project in build.ps1

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: utils/build.ps1 [hunk 1]
```diff
@@ -1237,11 +1237,23 @@ function Build-SPMProject {
...
+    Build {
+      $ActionForOutput = "Building"
+    }
+    Test {
+      $ActionForOutput = "Testing"
+    }
+    TestParallel {
+      $ActionForOutput = "Testing"
+    }
+  }
```
Comment (compnerd) [inline utils/build.ps1:1250]: I wonder if we can rewrite this as:

```suggestion
$ActionForOutput = switch ($Action) {
Build { "Building" }
Test { "Testing" }
TestParallel { "Testing" }
}
```

**Review by compnerd — APPROVED** [review]

File: utils/build.ps1 (hunk 1 above)
Comment (ahoppen) [reply utils/build.ps1:1250]: Ah, nice. Didn’t know that was possible.

**Comment by ahoppen** [comment]
@swift-ci Please smoke test

**Pull request merged**